include/pingcap/kv/RegionCache.h

```diff
@@ -195,7 +204,7 @@ class RegionCache
 
     Store getStore(Backoffer & bo, uint64_t id);
 
-    std::vector<uint64_t> getAllValidTiFlashStores(Backoffer & bo, const RegionVerID & region_id, const Store & current_store, const LabelFilter & label_filter);
+    std::pair<std::vector<uint64_t>, std::vector<uint64_t>> getAllValidTiFlashStores(Backoffer & bo, const RegionVerID & region_id, const Store & current_store, const LabelFilter & label_filter);
```

Comment:
Add some comments about the meaning of the return type.

Reply:
Done